models/pathology_tumor_detection/configs/multi_gpu_train.json

```diff
@@ -28,7 +28,10 @@
         "$import torch.distributed as dist",
         "$dist.is_initialized() or dist.init_process_group(backend='nccl')",
         "$torch.cuda.set_device(@device)",
-        "$monai.utils.set_determinism(seed=123)"
+        "$monai.utils.set_determinism(seed=123)",
+        "$import logging",
+        "$@train#trainer.logger.setLevel(logging.WARNING if dist.get_rank() > 0 else logging.INFO)",
```

Comment:
@yiheng-wang-nv I added a `RankFilter` for similar usage:

https://github.com/Project-MONAI/MONAI/blob/7f3d3fc79e0caa0953f801c903368e74a65a8773/monai/utils/dist.py#L182

Example to use
https://github.com/Project-MONAI/MONAI/blob/7f3d3fc79e0caa0953f801c903368e74a65a8773/monai/apps/auto3dseg/ensemble_builder.py#L560

Here it could be
```
"$@train#trainer.logger.addFilter(RankFilter())",
```

Please consider the suggestion as optional, and skip if needed.

Reply:
Thanks Mingxin, in this version I suggest to not using this feature (require latest MONAI commit). Feel free to add a feature request and we can address it in next version.